person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] Kindly set your id `attr_reader :id`,  id should only be able to be retrieved so only the Getter is defined